js/main.js

```diff
@@ -1,25 +1,66 @@
 $(document).ready(function () {
 
-    window.onload = function() {
 
-            var chart = new CanvasJS.Chart("chartContainer", {
-                animationEnabled: true,
-                title: {
-                    text: "Dosya Yüzdeleri"
-                },
-                data: [{
-                    type: "pie",
-                    startAngle: 240,
-                    yValueFormatString: "##0.00\"%\"",
-                    indexLabel: "{label} {y}",
-                    dataPoints: [
-                        {y: 100, label: ""},
+    window.onload = function() {
+        var analysis_info = $.get(
+            /**
+             * @var object data
+             */
+            OC.generateUrl('/apps/analysis_app/getinfo'), function (data) {
+                var allMimeTypes = [], i=0,totalMimeTypeSize=0;
+                $.each(data.mime_types, function (key,value) {
+                    totalMimeTypeSize = +totalMimeTypeSize + +value;
+                });
+                $.each(data.mime_types, function (key,value) {
+                    data.mime_types[key] = (value/totalMimeTypeSize)*100;
+                });
+                $.each(data.mime_types, function(key, value) {
+                    allMimeTypes[i] = {value: value, name: key};
+                    i++;
+                });
+                var mimeTypesChart = echarts.init(document.getElementById('chartContainer'));
+                var option = {
+                    title : {
+                        text: 'Dosya Tipi Yuzdeleri',

```

Comment:
we need to use translation in here. https://doc.owncloud.org/server/10.0/developer_manual/core/translation.html. We don't have to add the translations at this stage, but good to know this concept.